muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/reducers/base/locales.ts

```diff
@@ -4,15 +4,15 @@ import $ from '~/lib/jquery';
 import {ActionType} from "~/actions";
 
 export interface LocaleListType {
-  avaliable: {
+  AVAILABLE: {
     name: string,
     locale: string
   }[],
   current: string
 }
 
 export default function locales(state={
-  avaliable: $.makeArray($("#language-picker a").map((index: number, element: HTMLElement)=>{
+  AVAILABLE: $.makeArray($("#language-picker a").map((index: number, element: HTMLElement)=>{
```

Comment:
Why was it renamed into uppercase?...

Reply:
Must be a mistake.

Reply:
Aha yes, I replaced a bunch of typo stuff: "avaliable" is a typo.